runtime/src/main/kotlin/kotlin/native/internal/RuntimeUtils.kt

```diff
@@ -181,3 +181,40 @@ internal fun <T> listOfInternal(vararg elements: T): List<T> {
         result.add(elements[i])
     return result
 }
+
+
+@PublishedApi
```

Comment:
Btw, have you seen https://github.com/JetBrains/kotlin-native/blob/9a54689033c8ac1f9eb679066c0418f10454de68/runtime/src/main/kotlin/kotlin/native/internal/Boxing.kt#L35 ?

Reply:
Funny, will switch to those.